torch/_inductor/decomposition.py

```diff
@@ -502,7 +502,7 @@ def fmax(self: torch.Tensor, other: torch.Tensor) -> torch.Tensor:
     return torch.where(torch.isnan(other) | (other < self), self, other)
 
 
-@register_decomposition(aten.amax)
+@register_decomposition([aten.amax])
```

Comment:
This was one of the places where the new typing paid off. `register_decomposition` expects a list of `OperatorBase`/`OpOverloadPacket`. The previous typing of `_OpNamespace` returned `Any` from `__getattr__`, so the implicit call in `aten.amax` couldn't be determined to not be iterable. Once that `__getattr__` was explicitly typed to return `OpOverloadPacket`, typing failed here.